### PR TITLE
Add feature "project-words-filter"

### PR DESCRIPTION
**Pull request by Masaabu**
Remove or blur from the search page any projects that contain words you do not want to appear in the title.

### CODE REVIEW

**Comment by MaterArc** [comment]
Looks good so far! This seems like a nice feature, I did, however, make a couple of changes to make it even better! I noticed some more tiny things that I will try to fix later this evening and hopefully we can get this in as a version feature soon!

**Comment by MaterArc** [comment]
The filter input box should also support spaces in between items. When seeing that they are separated by commas, it should also allow spaces.

File: features/project-words-filter/data.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "description": "Hide projects that contain certain words on the search and explore pages.",
+    "credits": [
+        { "username": "Masaabu-YT", "url": "https://scratch.mit.edu/users/Masaabu-YT/" }
+    ],
+    "type": ["Website"],
+    "tags": ["New", "Featured"],
+    "dynamic": true,
+    "options": [
+        { "id": "blur", "name": "Blur thumbnails without erasing the project.", "type": 1 },
+        { "id": "filter-words", "name": "filter words (Split by comma)", "type": 0 }
```
[the remark this answers is not in the document]
Comment (Masaabu) [reply features/project-words-filter/data.json:12]: How about this?
image

**Comment by Masaabu** [comment]
Currently, the filter only works for project titles. Do we need the ability to filter by user name as well?

File: features/project-words-filter/data.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MaterArc) [reply features/project-words-filter/data.json:12]: > How about this? image

Looks pretty neat! A lot going on there, but that can be cleaned up later...

**Comment by mrfurretgo** [comment]
I was just thinking about making a feature like this! This was kinda done with Anti-Generic, but it would be great if they were able to be specified by the user.

Also, what if this feature makes it so the user can decide which keywords need their thumbnails blurred or the projects filtered? It would be a nice addition as well.

**Pull request closed without merging**